dns/.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: massa sc ci tests
+on: [push]
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+          cache: 'npm'
+
+      - name: Install
+        run: npm install
+
+      - name: Test
+        run: npm run test
```

Comment:
this workflow will not be executed. it must be in .github dir at the root of the repo